daemon/postprocess/PrePostProcessor.cpp

```diff
@@ -59,13 +59,25 @@ void PrePostProcessor::Run()
 
 	while (!IsStopped())
 	{
-		if (!g_Options->GetTempPausePostprocess() && m_queuedJobs)
+		if (g_Options->GetTempPausePostprocess())
+		{
+			// Postprocess is paused: just wait and loop
+			usleep(200 * 1000);
+			continue;
+		}
+
+		if (m_queuedJobs)
 		{
 			// check post-queue every 200 msec
 			CheckPostQueue();
+			usleep(200 * 1000);
+		}
+		else
+		{
+			// Wait until we get the stop signal or more jobs in the queue
+			std::unique_lock<std::mutex> lk(m_pauseMutex);
```

Comment:
Should probably be:

```C++
UniqueLock lk(m_pauseMutex);
```

Maybe also better to use name `guard` just like in all other places? Not sure about that though.

Consequently `m_pauseMutex` should be declared as `Mutex` instead of `std::mutex`.

And similarly in `FeedCoordinator.cpp/.h`.

Reply:
> Maybe also better to use name guard just like in all other places? Not sure about that though.

Yeah, I thought about that but as the type is std::unique_lock lock I left it like that because it better matches the real type.

Changed the rest.